.gitlab/test_nixlbench.sh

```
@@ -45,29 +45,19 @@ nvidia-smi topo -m || true
 
 DEFAULT_NB_PARAMS="--filepath /tmp --total_buffer_size 80000000 --start_block_size 16384 --max_block_size 16384 --start_batch_size 4 --max_batch_size 4"
 
-run_nixlbench_noetcd() {
-    args="$@"
-    ./bin/nixlbench $DEFAULT_NB_PARAMS $args
-}
-
 run_nixlbench_one_worker() {
     args="$@"
-    run_nixlbench_noetcd $args
+    ./bin/nixlbench $DEFAULT_NB_PARAMS $args
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)